src/opc-plc.csproj

```diff
@@ -1,4 +1,4 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">

```

Comment:
Pls update version.json so that we can create a new release

Reply:
Done. Set version to `2.9.9` there.